### DISCUSSION TITLE
Deployment process

### DISCUSSION THREAD

**Discussion by georgefst** [spiral-project/ihatemoney#1048] [General]
Is there any sort of policy or schedule as to when new releases are deployed?

I just realised that some changes I'd been waiting on were [released over three months ago](https://github.com/spiral-project/ihatemoney/releases/tag/5.2.0), but this isn't reflected at https://ihatemoney.org/.

**Comment by georgefst** [comment]
> I just realised that some changes I'd been waiting on were [released over three months ago](https://github.com/spiral-project/ihatemoney/releases/tag/5.2.0), but this isn't reflected at https://ihatemoney.org/.

There must have been a deployment at some point since this. As I now see the effects of e.g. `Hide "each" amount when there's only one recipient`.

**Comment by Glandos** [comment]
Hi !
Sorry for the late answer. As you may have guessed, there is nothing like a schedule or a policy. It's purely based on the spare time we have. When too much feature are stacking into master, we make a release.
However, note that the main instance at ihatemoney.org is usually up to date, but it is not updated automatically from last version.

Thanks for your interest in this project, I hope it's useful to you!

**Comment by ifeeltiredboss** [comment]
Maybe the releases could be somehow automated? Unfortunately, I do not have any skills to help with that. Just wanted to bump this discussion because of #1081 fix.

**Reply by almet** [reply]
I feel it's better to keep this a manual process, because it allows for less errors in general.
